pkg/trace/agent/agent_test.go

```diff
@@ -581,3 +586,165 @@ func formatTrace(t pb.Trace) pb.Trace {
 	}
 	return t
 }
+
+func BenchmarkThroughput(b *testing.B) {
+	if _, ok := os.LookupEnv("DD_TRACE_TEST_PRIVATE_DATA"); !ok {
+		b.SkipNow()
+	}
+
+	ddlog.SetupDatadogLogger(seelog.Disabled, "") // disable logging
+
```

Comment:
It doesn't have to be in that PR, but here is a proposal:
Instead of hardcoding the name of each file + the relative path, the env variable could simply point to a directory that contains payload dumps. Then the test just run the benchmark for each of these files.

That'd make modifications of the dumps much simpler and also not tight it to the structure of this external private repo.

Reply:
Done